scripts/cli/src/commands/helpers/identity_helper.ts

```diff
@@ -0,0 +1,34 @@
+import {nonces, sendTransaction} from "../util/init"
+import { topUpIdentityBalance } from "./balance_helper"
+import { IKeyringPair } from '@polkadot/types/types'
+import { ApiPromise } from "@polkadot/api"
+import { SigningItem } from "../types"
+
+export default async function createIdentities(api: ApiPromise, accounts: IKeyringPair[], alice: IKeyringPair) {
+    return await createIdentitiesWithExpiry(api, accounts, alice, []);
+};
+
+const createIdentitiesWithExpiry = async function(api: ApiPromise, accounts: IKeyringPair[], alice: IKeyringPair, expiries: SigningItem[]) {
+  let dids = [];
+
+  for (let i = 0; i < accounts.length; i++) {
+    let nonceObj = {nonce: nonces.get(alice.address)};
+    const transaction = api.tx.identity.cddRegisterDid(accounts[i].address, null, []);
+    await sendTransaction(transaction, alice, nonceObj);
+
+    nonces.set(alice.address, nonces.get(alice.address).addn(1));
+  }
+  
+  for (let i = 0; i < accounts.length; i++) {
+    const d: any = await api.query.identity.keyToIdentityIds(accounts[i].publicKey);
+    dids.push(d.raw.asUnique);
+  }
+  let did_balance = 1000 * 10**6;
+  for (let i = 0; i < dids.length; i++) {
+
+    await topUpIdentityBalance(api, alice, dids[i], did_balance);
```

Comment:
We could have a parameter `shouldTopUpIdentity` which specifies whether or not this part is executed or not.

Reply:
Ideally, as with the `cdd_provider` there would be a way for users to specify a different "funding" account which could default to `Alice` but be overriden by the user if they want. This account would then be used to fund both the identity (and possibly the new keys being generated as well optionally).

Reply:
okay, sounds plausible. I would add this to the changes to do as well.